docs/search/collaborative_editing_search_reference/invitation/updated_at_invitation_criterion.md

```diff
@@ -0,0 +1,23 @@
+---
+description: UpdatedAt Search Criterion
+---
+
+# UpdatedAt Criterion
+
+The `UpdatedAt` Search Criterion searches for invitations based on the date they were updated.
+
+## Arguments
+
+- `value` - date to be matched, provided as a DateTimeInterface object
+- `operator` - optional operator string (EQ, GT, GTE, LT, LTE)
```

Comment:
The same as for CreatedAt: Same as for CreatedAt criterion: https://github.com/ibexa/documentation-developer/pull/2721/files#diff-26f37ef92904dfa297749b074b2938ed6e63fb2bc79857eae5d1c607b4f8ed6f